src/components/Researcher/SaveResearcherData.tsx

```diff
@@ -10,23 +10,30 @@ interface StudyObject {
   sensors: Array<any>
 }
 export const fetchResult = async (authString, id, type, modal) => {
-  const baseUrl = "https://" + (!!LAMP.Auth._auth.serverAddress ? LAMP.Auth._auth.serverAddress : "api.lamp.digital")
+  const baseUrl = !!LAMP.Auth._auth.serverAddress ? LAMP.Auth._auth.serverAddress : "http://api.lamp.digital"
```

Comment:
`https`-only! Also, extraneous `http://` in the fallback case here will lead to `fetch("http://http://api.lamp.digital/....")`